kubernetes-ts-stateful/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "k8s-nginx",
+  "version": "0.1.0",
+  "devDependencies": {
+    "@types/node": "^9.3.0"
+  },
+  "dependencies": {
+    "@pulumi/kubernetes": "^0.16.0",
+    "@pulumi/pulumi": "^0.15.0"
```

Comment:
Move this to `0.16.0`, and maybe Kubernetes to `0.17.0`?

Reply:
We probably just want to be using `latest` here, as with our other examples right?  That way we don't have to keep updating these examples every time we bump (minor) package versions?